Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -59,7 +59,7 @@ namespace Multiplayer
     AZ_CVAR(AZ::CVarFixedString, editorsv_serveraddr, AZ::CVarFixedString(LocalHost), nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "The address of the server to connect to");
     AZ_CVAR(AZ::CVarFixedString, editorsv_rhi_override, "", nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
         "Override the default rendering hardware interface (rhi) when launching the Editor server. For example, you may be running an Editor using 'dx12', but want to launch a headless server using 'null'. If empty the server will launch using the same rhi as the Editor.");
-    AZ_CVAR(uint16_t, editorsv_max_connection_attempts, 5, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
+    AZ_CVAR(uint16_t, editorsv_max_connection_attempts, 15, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
How about we set this CVar value through the command line that launches the test?

Reply:
@jromnoa Would you happen to know how to inject a command line argument to an Editor Python test.
Looking at the git blame, it looks like you added support for an [extra_cmdline_args](https://github.com/o3de/o3de/blame/development/Tools/LyTestTools/ly_test_tools/o3de/multi_test_framework.py#L826-L828) option?

Specifically, the [test_Multiplayer_BasicConnectivity_Connects](https://github.com/o3de/o3de/blob/development/AutomatedTesting/Gem/PythonTests/Multiplayer/TestSuite_Main.py#L38-L39) test would need the command line argument injected.

Reply:
Yeah, adding it in pytest, or even an AutomatedTesting setreg? I like your suggesting for setting it right inside of pytest, I'm sure I've done that before... searching for the the method now.

Reply:
How's this? It a little different than setting the cvar via commandline, and instead just setting it before the test runs. Thoughts?

Reply:
That is good with me.